init/minimal_eessi_env

```diff
@@ -1,7 +1,7 @@
 # define minimal EESSI environment, without relying on external scripts
 
 export EESSI_CVMFS_REPO="/cvmfs/pilot.eessi-hpc.org"
-export EESSI_PILOT_VERSION="2021.12"
+export EESSI_PILOT_VERSION="${BUILD_EESSI_VERSION:=2021.12}"
```

Comment:
How about something more direct like `$EESSI_PILOT_VERSION_OVERRIDE`?

Reply:
You mean something like

`export EESSI_PILOT_VERSION="${EESSI_PILOT_VERSION_OVERRIDE:=2021.12}"`